### ISSUE TITLE
[Sourcing-2][Accessibility: ACC-264.1] Button Menu: 2 UI types conveyed

### ISSUE THREAD

**Issue by JyotiKumari1507** [SAP/fundamental-ngx#11254]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
Component having issue: https://sap.github.io/fundamental-ngx/#/core/shellbar

productMenuControl  of <fd-product-menu>  is accessed 2 UI types, Toggle button and menu is conveyed which is incorrect. Single UI type based on the functionality of the UI should be conveyed.

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
All latest version

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Using Screen reader can check

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations. 
Yes we have got this bug from Accessibility expert only.

#### Did you check the documentation and the API?
Yes

**Comment by nikolay-kolarov** [comment]
Hi @JyotiKumari1507,
Hi @droshev,

The button has aria-haspopup="menu" to indicate that on press a menu will appear.
And this is from where the announcement comes.
It is the same as implementation in UI5: https://ui5.sap.com/#/entity/sap.f.ProductSwitch/sample/sap.f.sample.ShellBarProductSwitch
So I think we can close this.

Best Regards,
Nikolay

**Comment by JyotiKumari1507** [comment]
@droshev Attaching screenshot which clearly state that screen reader is reading shellbar mebu as **Toogle Toggle** button. Which is wrong.. Please re-open. Still any query please let me know we can have call
<img width="452" alt="image" src="https://github.com/SAP/fundamental-ngx/assets/157268666/913ad19f-cebb-4829-8d3c-fbafe3170129">

**Comment by nikolay-kolarov** [comment]
Hello @JyotiKumari1507,

I am not sure what exactly was discussed and I do not get what is the problem here.

We have two things:
- A button that is used to open a menu:
![image](https://github.com/SAP/fundamental-ngx/assets/17043004/ea8fa646-0542-432f-b821-6c01f7f600fa)
It is defined as a standard HTML button with aria-pressed attribute and aria-haspopup=menu. And the expected output is exactly as shown on the picture (toggle button ... has popup ... menu .. to toggle press). You can test this with simple HTML `<button aria-pressed="true" aria-haspopup="menu">More info</button>`.
- The menu which is opened: It is a real menu designed like this:
![image](https://github.com/SAP/fundamental-ngx/assets/17043004/04ba7259-d277-493f-b6b8-7580707c7827)

So I do not get what is the problem here. Please explain what exactly you think is wrong and maybe give a sample what you expect as output.

Thanks,
Nikolay

**Comment by JyotiKumari1507** [comment]
@nikolay-kolarov I also see similar this on code. But as per accessibility team, menu is reading out as **Toggle Toogle** instead of menu by screen reader. I have already asked for recording from them so lets wait for recording by Accessibility team

**Comment by MeghanaVernekar** [comment]
Hello @JyotiKumari1507,
 I am unable to login with the old URL which I had used for testing: https://qa.cobalt.ariba.com/emc-ui-api/.
 
 Can you please provide me the updated URL to reproduce the above issue and provide the recording.

Regards,
Meghana.

**Comment by divyamalhotra488** [comment]
> Hello @JyotiKumari1507, I am unable to login with the old URL which I had used for testing: https://qa.cobalt.ariba.com/emc-ui-api/.
> 
> Can you please provide me the updated URL to reproduce the above issue and provide the recording.
> 
> Regards, Meghana.

Hi @MeghanaVernekar, provided you the new url and login access for the same.

**Comment by MeghanaVernekar** [comment]
https://github.com/SAP/fundamental-ngx/assets/172006721/0bd09aa7-357f-4113-84c2-0cb23ad66b02

Hello Colleagues,

Attached video as requested, and issue no more exists. 

<img width="960" alt="EMC" src="https://github.com/SAP/fundamental-ngx/assets/172006721/1ea5f7f1-faa6-4a2e-830e-ef804836c02d">

Regards,
Meghana.

**Comment by droshev** [comment]
Can you open a new ticket and creating a stackblitz with the issue?

**Comment by divyamalhotra488** [comment]
Hi @droshev, as mentioned by @MeghanaVernekar the issue no longer exists.
